File: src/my_project/easy_problems/from51to100/shuffle_array.py
from typing import List

class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        temp = list()

        for i in range(n):
            temp.append(nums[i])
            temp.append(nums[n+i])
        return temp

solution  = Solution()
print(solution.shuffle([2,5,1,3,4,7], 3))

File: my_java_projects/src/easy_problems/from_1_to_50/ShuffleArray.java
package easy_problems.from_1_to_50;

public class ShuffleArray implements Numbers{
    public int[] shuffle(int[] nums, int n) {

        int lenNums = nums.length;
        int halfLenNums = (int)nums.length/2;
        int[] solution = new int[lenNums];

        for (int i = 0; i<halfLenNums; i++){
            solution[i*2] = nums[i];
            solution[i*2+1] = nums[halfLenNums + i];
        }


        return solution;

    }

    @Override
    public void returnNumbers() {
        System.out.println("This class does not return numbers.");
    }
}
